packages/services/server/src/index.ts

```diff
@@ -128,7 +128,12 @@ export async function main() {
     },
   );
 
-  server.setErrorHandler(supertokensErrorHandler());
+  let errorHandler = supertokensErrorHandler();
+  server.setErrorHandler((err, req, res) => {
+    // DOTAN: Sorry about "as any" here, but it's the only way to make it work.
```

Comment:
@n1ru4l @kamilkisiela wasn't sure about this one. We are using v15 of supertokens-node and the latest is v20, seems like a whole different adventure. 

@n1ru4l How can I test/trigger this error handler during development to make sure it's working?